### PR TITLE
Chore: Don't run partial build for tags or release

### PR DESCRIPTION
**Pull request by nschonni**
From guidance in http://help.appveyor.com/discussions/questions/261-only-deploy-when-theres-a-new-tag#comment_39426826

### CODE REVIEW

**Comment by nschonni** [comment]
Hmm, this doesn't seem like it likes it since there is no AppVeyor build triggered :(

File: appveyor.yml [hunk 1]
```diff
@@ -79,6 +79,12 @@
         appveyor_repo_tag: true       # deploy on tag push only
 
 -
+  branches:
+    except:
+      - release
+
+  skip_non_tags: true
```
Comment (xzyfer) [inline appveyor.yml]: You'd want to skip tags here, not non-tags.

**Comment by xzyfer** [comment]
You're only building tags now which is the inverse desired behaviour.

File: appveyor.yml [hunk 2]
```diff
@@ -79,6 +81,12 @@
         appveyor_repo_tag: true       # deploy on tag push only
 
 -
+  branches:
+    except:
+      - release
+
+  skip_non_tags: false
```
Comment (xzyfer) [inline appveyor.yml]: Is this redundant?

Comment (xzyfer) [inline appveyor.yml]: Is there way to explicitly tell AppVeyor not to build tags?

Comment (nschonni) [inline appveyor.yml]: I added the `true` section in the tag config above, but maybe that's still not right

Comment (xzyfer) [inline appveyor.yml]: I would think you only need here if the configs cascade which I don't believe they do. Try removing this and see if AppVeyor still builds the branch.

**Comment by xzyfer** [comment]
FWIW you should work on a fork otherwise you trigger a lot of builds.

**Pull request closed without merging**